compiler/noirc_frontend/src/parser/parser.rs

```diff
@@ -369,10 +370,41 @@ fn optional_type_annotation<'a>() -> impl NoirParser<UnresolvedType> + 'a {
         .map(|r#type| r#type.unwrap_or_else(UnresolvedType::unspecified))
 }
 
+/// import_visibility: 'pub(crate)'? 'pub'? ''
```

Comment:
```suggestion
/// import_visibility: 'pub(crate)' | 'pub' | %empty
```
The grammar with `?` would accept `pub(crate) pub`